man/fi_fabric.3.md

```diff
@@ -157,6 +157,17 @@ A fabric identifier.
 
 The name of the underlying fabric provider.
 
+For debugging purposes, environment variables can be used to control which
```

Comment:
*shrug* I suppose administrative reasons would be a valid use of this sort of functionality.  I was mainly trying to steer people away from its general use because for applications it's more robust to put real provider selection logic in the app's code instead of relying on environment variables being set everywhere (we've both seen this in OMPI, for example).  I'll change it to "For debugging and and administrative purposes".